src/CommonLib/LDAPUtils.cs

```diff
@@ -378,13 +472,25 @@ public IEnumerable<string> DoRangedRetrieval(string distinguishedName, string at
             if (searchRequest == null)
                 yield break;
 
+            var backoffDelay = MinBackoffDelay;
+            var retryCount = 0;
+            
             while (true)
             {
                 SearchResponse response;
                 try
                 {
                     response = (SearchResponse) conn.SendRequest(searchRequest);
                 }
+                catch (LdapException le) when (le.ErrorCode == 51 && retryCount < MaxRetries)
```

Comment:
Not a huge deal but it might be worth extracting ldap error codes like this into a constant to make `51` a little more understandbale.